scripts/run_backend_tests.py

```diff
@@ -659,6 +695,25 @@ def check_coverage(
     process = subprocess.run(
         cmd, capture_output=True, encoding='utf-8', env=env,
         check=False)
+
+    not_covered_lines = []
+    is_first_line = True
```

Comment:
Try not to start booleans with a verb. Maybe something_is_first_line (specify what "something" is).

Reply:
Also do you even need this (couldn't you just check whether not_covered_lines is empty)?

Reply:
@seanlip We need this variable in the case where all the files have 100% coverage. In such a scenario, there is no need to add the heading.

Reply:
@seanlip i have removed the boolean variable in new updated code. Thanks.